index.html

```diff
@@ -5,11 +5,76 @@
+                WELCOME TO BLABLA RESTAURANT
+                <img src=" ./images/PngItem_303341.png" alt="background image">
+                <img src="./images/wine-glass-red-wine-wine-cocktail-champagne-drink-wine-d3d72936183c1a59da94eb28e13d3e17.png"
+                    alt="background image 2">
+            </h1>
+        </section>
+        <section id="menu" class="container-fluid px-4">
+            <div class="h2-wrapper">
+                <h2>Menu
+                </h2>
+                <div class="line"></div>
+            </div>
+            <div class="g-2 row row-cols-1 row-cols-sm-2 row-cols-md-4">
+                <div class="col">
+                    <div
+                        class="card-menu d-flex align-items-center justify-content-around justify-content-sm-start flex-sm-column">
+                        <img class="menu-icons" src="./images/lunch-3.png" alt="lunch icon">
+                        <h3 class="my-3">Lunch</h3>
+                        <p class="col-6 col-sm-12">Served from 10:00 am to 4:00 pm</p>
+                    </div>
+                </div>
+                <div class="col">
+                    <div
+                        class="card-menu d-flex align-items-center justify-content-around justify-content-sm-start flex-sm-column">
+                        <img class="menu-icons" src="./images/dinner-3.png" alt="lunch icon">
+                        <h3 class="my-3">Dinner</h3>
+                        <p class="col-6 col-sm-12">Served from 4:00 pm to 11:00 pm</p>
+                    </div>
+                </div>
+                <div class="col">
+                    <div
+                        class="card-menu d-flex align-items-center justify-content-around justify-content-sm-start flex-sm-column">
+                        <img class="menu-icons" src="./images/bar-3.png" alt="lunch icon">
+                        <h3 class="my-3">Bar</h3>
+                        <p class="col-6 col-sm-12">Wine, vodka, whiskey</p>
+                    </div>
+                </div>
+                <div class="col">
+                    <div
+                        class="card-menu d-flex align-items-center justify-content-around justify-content-sm-start flex-sm-column">
+                        <img class="menu-icons" src="./images/hot-3.png" alt="lunch icon">
+                        <h3 class="my-3">Hot</h3>
+                        <p class="col-6 col-sm-12">Coffee, tea</p>
+                    </div>
+                </div>
+            </div>
+            <button class="btn btn-orange">Make Reservation</button>
+        </section>
+        <section id="team" class="container gap-4">
+            <div class="h2-wrapper">
+                <h2>Our Team</h2>
+                <div class="line"></div>
+            </div>
+            <div class="gy-4 gx-4 px-3 row row-cols-1 row-cols-md-2 justify-content-center" id="team-members"></div>
+        </section>
```

Comment:
@brytebee Same comment as above.
What is the difference between 3 or 4 or 5 articles in terms of coding? I can copy/paste the same card to the infinite amount.

Reply:
@damdafayton As stated in the summary, everything I have required is stated in the requirement and because it is easy that is why it should be done to the letter to show your keen design skills to keep to the tiniest detail.

![image](https://user-images.githubusercontent.com/27709832/152162333-4e7c22d0-f123-4d8b-9266-3f5e2fff0a72.png)

Reply:
@brytebee Those things are not on our ["requirements list"](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)
It would have been much helpful and prevented time loss, if they put that information on the activity page.